### DISCUSSION TITLE
How do i export my vercel / next js analytics data from the vercel dashboard?

### DISCUSSION THREAD

**Discussion by rcolepeterson** [vercel/next.js#60790] [Help]
### Summary

How do i export my Vercel / Next JS analytics data?
I cant find the button.

I need to collect the data and share it with others.

**Comment by RizBizKits** [comment]
Hey! This is requested feature and is on our roadmap to ship (I am unable to confirm an ETA). 
For now, both Speed Insights and Web Analytics, are viewable exclusively on the Vercel Dashboard.
